serverless-fraud-detection/docs/modules/ROOT/pages/serverless-fraud-detection.adoc

```diff
@@ -0,0 +1,555 @@
+= Serverless Fraud Detection with AWS and {hazelcast-cloud}
+:description: In this tutorial, you'll build a serverless solution for detecting fraudulent transactions using AWS Lambda, Hazelcast Viridian, and Node.js.
+:page-layout: tutorial
+:page-product: cloud
+:page-categories: Caching, AWS Lambda
+:page-est-time: 30 mins
+:page-lang: node
+:url-code-sample: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/serverless-fraud-detection
+
+{description}
+
+== Context
+
+Every time your credit or debit card initiates a transaction, the underlying payment system validates the transaction with a series of checks. Some checks are simple, such as making sure that you have enough funds. Other checks are more advanced, such as filtering the history of card payments to identify a personalized pattern and validating the given transaction against that pattern.
+
+In this tutorial, you'll build an application that validates transactions made at airports, using the following process:
+
+. Compare the time between two transactions. A is the previous transaction and B is the current transaction.
+. Calculate the distance between the airports at which the transactions were made, using the link:https://en.wikipedia.org/wiki/Haversine_formula[haversine formula].
```

Comment:
```suggestion
. Calculate the distance between the airports where the transactions were made using the link:https://en.wikipedia.org/wiki/Haversine_formula[haversine formula].
```

Reply:
The comma is necessary to make it clear that the transaction were not made using the haversine formula.